plugins/openvr/src/OpenVrHelpers.cpp

```diff
@@ -108,9 +108,11 @@ QString getVrSettingString(const char* section, const char* setting) {
     return result;
 }
 
+bool isHMDInErrorState = false;
+
 vr::IVRSystem* acquireOpenVrSystem() {
     bool hmdPresent = vr::VR_IsHmdPresent();
-    if (hmdPresent) {
+    if (hmdPresent && (!isHMDInErrorState)) {
```

Comment:
excessive parentheses
```suggestion
    if (hmdPresent && !isHMDInErrorState) {
```